build.sh

```diff
@@ -71,7 +69,6 @@ case "$COMMAND" in
     -scheme "${SCHEME}" \
     -sdk "${IOS_SDK}" \
     -destination "${IOS_DESTINATION_PHONE}" \
-    -destination "${IOS_DESTINATION_PAD}" \
```

Comment:
Is this change intentional?

Reply:
Yes for some reason xcodebuild10.1 could not run with multiple destinations. Is it important to keep?

Reply:
It's fine I guess